client/src/pages/Submission/Submission.tsx

```diff
@@ -1,7 +1,176 @@
+import React, { useState, useEffect } from 'react';
+import { Grid, Button, Typography, Box, Snackbar, IconButton } from '@material-ui/core';
+import CloudUploadIcon from '@material-ui/icons/CloudUpload';
+import useStyles from './useStyles';
+import axios from 'axios';
+import DeleteOutlineIcon from '@material-ui/icons/DeleteOutline';
+import CloseIcon from '@material-ui/icons/Close';
+
 export default function Submission(): JSX.Element {
+  const classes = useStyles();
+  // For loading file to state and allowing preview
+  const [inputFiles, setInputFiles] = useState<File>();
+  // For generating the preview img string
+  const [preview, setPreview] = useState<string>();
+  // For snackbar notifications opening/closing
+  const [open, setOpen] = useState(false);
+  //for the snackbar message
+  const [message, setMessage] = useState<string>();
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  //   for generating image preview
+  useEffect(() => {
+    if (inputFiles) {
+      const reader = new FileReader();
+      reader.onloadend = () => {
+        setPreview(reader.result as string);
+      };
+      reader.readAsDataURL(inputFiles);
+    } else {
+      setPreview('');
+    }
+  }, [inputFiles]);
+
+  // For getting our image list into state
+  const handleImageChange = ({ currentTarget: { files } }: React.ChangeEvent<HTMLInputElement>) => {
+    if (files && files.length) {
+      setInputFiles(files[0]);
+    }
+  };
+  // For deleting the previewed image if the user decides to upload something different
+  const handleImageDelete = () => {
+    setPreview('');
+    setInputFiles(undefined);
+  };
+
+  const handleUpload = () => {
+    const data = new FormData();
+    // if file selected
+    if (inputFiles) {
+      data.append(
+        //this is gonna be the name of the response object. needs to match what's in the function(routes/api/profile.js line 38)
+        'multiImage',
+        inputFiles,
+        inputFiles.name,
+      );
+      // our post route. Might need to change depending on the final location in the app
+      //also i have questions about the boundary property. In plain react you can set it to data._boundary but here that's not allwoed
+      axios
```

Comment:
looks like you're introducing axios to the repo - you may want to sync with your team to make sure this is the practice you all want to be consistent on